clang/test/ClangScanDeps/modules-full-named-modules.cppm

```diff
@@ -0,0 +1,347 @@
+// This checks that clang-scan-deps properly outputs named module dependencies 
+// when using the the scanning output format 'experimental-full'.
+//
+// See commit 72304.
+// UNSUPPORTED: target={{.*}}-aix{{.*}}
+//
+// The slash direction in linux and windows are different.
+// UNSUPPORTED: system-windows
```

Comment:
We typically deal with separator differences by normalizing them to `/` like so:
```shell
// RUN: cat %t/result.json | sed 's:\\\\\?:/:g' | FileCheck -DPREFIX=%/t %s
```
Would something like that work for this test?

I see most of your matches are fairly simple (e.g. `[[PREFIX]]/M.cppm`) and there are no complex search paths involved, so maybe matching with `{{.*}}M.cppm` would do the trick too.